app/main.py

```diff
@@ -2,4 +2,70 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords if coords is not None else [0, 0]
```

Comment:
```suggestion
        self.coords = self.coords = coords or [0, 0]
```
better to use `or` in this case